tic-tac-toe-ts/methods.ts

```diff
@@ -0,0 +1,297 @@
+import * as readlineSync from "readline-sync";
```

Comment:
- you have clubbed all your methods in one file
- this isn't that much better than putting all your code in the app.js file

Reply:
- you should create separate files for separate use cases
- like for example a `PlayerUtil` class for player related action and `GameUtil`
- ask Ritik how to use a util class with static methods